packages/shorebird_cli/lib/src/commands/release_new/release_new_command.dart

```diff
@@ -0,0 +1,420 @@
+import 'dart:async';
+
+import 'package:mason_logger/mason_logger.dart';
+import 'package:meta/meta.dart';
+import 'package:scoped/scoped.dart';
+import 'package:shorebird_cli/src/code_push_client_wrapper.dart';
+import 'package:shorebird_cli/src/command.dart';
+import 'package:shorebird_cli/src/commands/release_new/aar_releaser.dart';
+import 'package:shorebird_cli/src/commands/release_new/android_releaser.dart';
+import 'package:shorebird_cli/src/commands/release_new/release_type.dart';
+import 'package:shorebird_cli/src/commands/release_new/releaser.dart';
+import 'package:shorebird_cli/src/config/config.dart';
+import 'package:shorebird_cli/src/extensions/arg_results.dart';
+import 'package:shorebird_cli/src/logger.dart';
+import 'package:shorebird_cli/src/platform/platform.dart';
+import 'package:shorebird_cli/src/shorebird_env.dart';
+import 'package:shorebird_cli/src/shorebird_flutter.dart';
+import 'package:shorebird_cli/src/third_party/flutter_tools/lib/flutter_tools.dart';
+import 'package:shorebird_code_push_client/shorebird_code_push_client.dart';
+import 'package:shorebird_code_push_protocol/shorebird_code_push_protocol.dart';
+
+typedef ResolveReleaser = Releaser Function(ReleaseType releaseType);
+
+/// {@template release_command}
+/// Creates a new app release for the specified platform(s).
+/// {@endtemplate}
+class ReleaseNewCommand extends ShorebirdCommand {
+  /// {@macro release_command}
+  ReleaseNewCommand({ResolveReleaser? resolveReleaser}) {
+    _resolveReleaser = resolveReleaser ?? getReleaser;
+    argParser
+      ..addOption(
+        'target',
+        abbr: 't',
+        help: 'The main entrypoint file of the application.',
+      )
+      ..addOption(
+        'flavor',
+        help: 'The product flavor to use when building the app.',
+      )
+      ..addOption(
+        'build-number',
+        help: 'The build number of the aar',
```

Comment:
I wonder if the term `aar` makes sense here?

Reply:
Good call, lifted the `flutter build` help text